src/games/rota/Field.tsx

```diff
@@ -22,18 +23,38 @@ const playerColors = {
 export class Field extends React.Component<IFieldProps, {}> {
   render() {
     return (
-      <svg width="100%" height="100%" viewBox="-250 -250 500 500" className={css.RotaField} pointerEvents="visible">
-        <g strokeWidth="4" stroke="#cccccc">
-          {this._drawLinesAndCircle()}
-        </g>
-        <g>{this._placePieces()}</g>
-      </svg>
+      <div style={{ textAlign: 'center' }}>
+        <svg
+          width={`${this.props.boardSize}%`}
```

Comment:
You have BOARD_SIZE and props.boardSize, what is the difference?

Reply:
well, when the game gets over, it is called with 50% of the original size.

Reply:
I see, can you rename one of those variables so they don't seem to mean the same thing?

You could rename the props to be boardSizePerc for percentage, or the other one for SVG_BOARD_SIZE